### DISCUSSION TITLE
Any way to toggle window titles with a key bind?

### DISCUSSION THREAD

**Discussion by moshpirit** [i3/i3#6347] [Q&A] [answered]
I don't like using titles because it normally doesn't apport me any useful information, and it takes space from my small laptop screen. But sometimes, it's good to know the name of a window, so I thought that it would be nice to be able to toggle this view by pressing a key bind, I don't know if this is possible right now, but I didn't find anything in the guide.

**Comment by orestisfl** [comment]
See https://i3wm.org/docs/userguide.html#_changing_border_style

**Reply by moshpirit** [reply]
Thanks for replying! I did, but it doesn't work with tabbed windows, only with tiling windows; and cannot be toggled since we need two different key bindings.

**Reply by okraits** [reply, marked as answer]
https://i3wm.org/docs/userguide.html#default_border

`The title bar is always visible in stacking and tabbed layouts, and this cannot be changed through configuration.`

**Reply by moshpirit** [reply]
I see, thanks a lot for replying!
